src/data/pokemon_graphics/enemy_mon_elevation.h

```diff
@@ -131,5 +131,7 @@ const u8 gEnemyMonElevation[NUM_SPECIES] =
     // Misc Forms
     [SPECIES_GIRATINA_ORIGIN] = 7,
     [SPECIES_THUNDURUS_THERIAN] = 7,
+	// Eternamax
```

Comment:
Is there really a need to add a comment for this instead of just leaving the entry under Therian Thundurus'?

Reply:
I never ended up editing this file properly when I did the form expansion months ago, but I was intending on syncing this up to how the other related files organize things too with species or group-specific heading comments.

Reply:
> but I was intending on syncing this up to how the other related files organize things too with species or group-specific heading comments.

Isn't that what the "Misc Forms" comment above Giratina's Origin Form is supposed to do? To organize the list of misc. form of existing species in this file? :eyes: ...

I'm just saying that adding an "// Eternatus" comment looks unnecessary to me instead of just packing it with the rest of the "Misc Forms" under the "Misc Forms" comment that is already there.

Reply:
Oops, I was the one that made this change before. Probably didn't further separate what's under the Misc Forms because there were only two entries so yeah this change is probably not necessary.